lib/sinon/stub-non-function-property.js

```diff
@@ -3,15 +3,13 @@
 var valueToString = require("./util/core/value-to-string");
 var hasOwnProperty = Object.prototype.hasOwnProperty;
 
-function stubNonFunctionProperty(object, property, value) {
+function stubNonFunctionProperty(object, property) {
```

Comment:
> The replacement for this usage is not documented

The practice of using `sandbox.stub` for [stubbing out properties has been documented for awhile](http://sinonjs.org/releases/v2.3.8/sandbox/#sandboxstub), admittedly, not very well.